api/rds.proto

```diff
@@ -566,6 +566,9 @@ message RedirectAction {
   // The path portion of the URL will be swapped with this value.
   string path_redirect = 2;
 
+  // [#not-implemented-hide:] The scheme portion of the URL will be swapped with this value.
+  string scheme_redirect = 4;
```

Comment:
nit: Can you try to keep this roughly inline w/ the fields? So move below?

Also, given that realistically this is only going to be HTTP and HTTPS, I wonder if we should make this an enum vs. a string? @htuch @wora WDYT?

Reply:
+1 on enum (or even boolean, do you ever want to downgrade from https?).